data/genomics/sarscov2/genome/baits.interval_list

```diff
@@ -0,0 +1,7 @@
+@HD	VN:1.6	SO:coordinate
+@SQ	SN:MT192765.1	LN:29829	M5:c95f3e5592d0ad9974e41e7f0ea14eb0	UR:file:/Users/kevin/Documents/nfcore/modules/tests/data/fasta/sarscov2/GCA_011545545.1_ASM1154554v1_genomic.fna
```

Comment:
Does the UR need to be changed here to a raw github URL?

Reply:
done! :)